flexmeasures/data/schemas/times.py

```diff
@@ -15,6 +15,42 @@ class DurationValidationError(FMValidationError):
     status = "INVALID_PERIOD"  # USEF error status
 
 
+class NewDurationField(MarshmallowClickMixin, fields.Str):
+    """Field that deserializes to a ISO8601 Duration
```

Comment:
All docstrings in this field still need to be updated. The idea behind this new field is that we realized we can store nominal sensor resolutions in our database by simply passing the original ISO string. For example, "P1D" or "P1M", which we find useful resolutions for reporter sensors. This field validates the ISO string, but the deserialization step returns the original string, to be passed to the Sensor's `__init__` method.

Dealing with nominal resolutions stored in our database is a separate issue (which should be created).

Reply:
Updated the docstrings and created #772 on how nominal resolutions should be handled.